ui/panels-plugin/src/plugins/time-series-chart/time-series-chart-model.ts

```diff
@@ -20,11 +20,22 @@ import { ThresholdOptions } from '../../model/thresholds';
  */
 export interface TimeSeriesChartOptions {
   queries: TimeSeriesQueryDefinition[];
-  show_legend?: boolean;
+  legend: LegendOptions;
   unit?: UnitOptions;
   thresholds?: ThresholdOptions;
 }
 
+export const DEFAULT_LEGEND: LegendOptions = {
+  show: false,
+  position: 'bottom',
+};
+
+export const LEGEND_POSITIONS = ['bottom', 'right'] as const;
+
+export type LegendPositionOptions = {
+  position: typeof LEGEND_POSITIONS[number];
```

Comment:
You might consider aliasing this `typeof` type, cause I could see it being useful:

```ts
export type LegendPosition = typeof LEGEND_POSITIONS[number];
```